packages/components/src/components/contract-card/contract-card-items/multiplier-card-body.jsx

```diff
@@ -0,0 +1,171 @@
+                </ContractCardItem>
+                {has_progress_slider && is_mobile && !is_sold && (
+                    <ContractCardItem className='dc-contract-card__date-expiry'>{progress_slider}</ContractCardItem>
+                )}
+                <div className='dc-contract-card__limit-order-info'>
+                    <ContractCardItem header={TAKE_PROFIT} className='dc-contract-card__take-profit'>
+                        {take_profit ? <Money amount={take_profit} currency={currency} /> : <strong>-</strong>}
+                    </ContractCardItem>
+                    <ContractCardItem header={STOP_LOSS} className='dc-contract-card__stop-loss'>
+                        {stop_loss ? (
+                            <React.Fragment>
+                                <strong>-</strong>
+                                <Money amount={stop_loss} currency={currency} />
+                            </React.Fragment>
+                        ) : (
+                            <strong>-</strong>
+                        )}
+                    </ContractCardItem>
+                    {(is_valid_to_sell || is_valid_to_cancel) && (
+                        <ToggleCardDialog
+                            addToast={addToast}
+                            connectWithContractUpdate={connectWithContractUpdate}
+                            contract_id={contract_info.contract_id}
+                            current_focus={current_focus}
+                            error_message_alignment={error_message_alignment}
+                            getCardLabels={getCardLabels}
+                            getContractById={getContractById}
+                            is_valid_to_cancel={is_valid_to_cancel}
+                            onMouseLeave={onMouseLeave}
+                            removeToast={removeToast}
+                            setCurrentFocus={setCurrentFocus}
+                            should_show_cancellation_warning={should_show_cancellation_warning}
+                            status={status}
+                            toggleCancellationWarning={toggleCancellationWarning}
+                        />
+                    )}
+                </div>
+            </div>
+            <ContractCardItem
+                className='dc-contract-card-item__total-profit-loss'
+                header={TOTAL_PROFIT_LOSS}
+                is_crypto={isCryptocurrency(currency)}
+                is_loss={+total_profit < 0}
+                is_won={+total_profit > 0}
+            >
+                <Money amount={total_profit} currency={currency} />
+                <div
+                    className={classNames('dc-contract-card__indicative--movement', {
+                        'dc-contract-card__indicative--movement-complete': is_sold,
+                    })}
+                >
+                    {status === 'profit' && <Icon icon='IcProfit' />}
+                    {status === 'loss' && <Icon icon='IcLoss' />}
```

Comment:
```suggestion
                    {<Icon icon={status === 'profit' ? 'IcProfit' : 'IcLoss' }
```

Reply:
unfortunately, status can be also null, that's why in such case it will show the icon for 'lost'  😞